client/src/app/shared/components/hook-form-pf-fields/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./hook-form-pf-group-controller";
 export * from "./hook-form-pf-text-input";
 export * from "./hook-form-pf-text-area";
+export * from "./hook-form-pf-datepicker";
```

Comment:
Will need to drop these bits because...I'm not ready to write reusable components.